src/main/java/com/wearsafe/service/jdbcdemo.java

```diff
@@ -0,0 +1,31 @@
+package com.wearsafe.service.impl;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.Statement;
+ 
+public class jdbcdemo {
+    public static void main(String[] args) {
+        Connection connection = null;
+        Statement stmt = null;
+        try
+        {
+            Class.forName("com.mysql.jdbc.Driver");
+            connection = DriverManager
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer

**Problem**
This line of code might contain a resource leak. Resource leaks can cause your system to slow down or crash.

**Fix**
Consider closing the following resource: *connection*. Currently, there are execution paths that do not contain closure statements. Either a) close *connection* in a try-finally block or b) close the resource by declaring *connection* in a try-with-resources block.

**More info**
[View resource management guidelines at oracle.com](https://www.oracle.com/technetwork/articles/java/trywithresources-401775.html) (external link).